bot/services/ChatBot.cs

```diff
@@ -334,6 +365,12 @@ await foreach (var chatResponse in streamingCall.ResponseStream.ReadAllAsync(can
 
     protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
     {
+        // verify authorization
+        if (!this.IsAuthorized(turnContext))
+        {
+            throw new Exception("the activity was not authorized to use this bot.");

```

Comment:
will this return a 403 later or how does this work?

Reply:
This is gRPC, so definitely not a 403. This raises to the AdapterWithErrorHandler. How we decide to handle that has been moved outside of the scope of this engagement, but you could imagine they might decide to respond to the user in some different way.